hack/generator/pkg/codegen/pipeline/implement_chained_conversion_interface.go

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package pipeline
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+
+	"github.com/Azure/azure-service-operator/hack/generator/pkg/astmodel"
+	"github.com/Azure/azure-service-operator/hack/generator/pkg/conversions"
+	"github.com/Azure/azure-service-operator/hack/generator/pkg/functions"
+)
+
+// ImplementConvertibleSpecInterfaceStageId is the unique identifier for this pipeline stage
+const ImplementConvertibleSpecInterfaceStageId = "implementConvertibleSpecInterface"
+
+func ImplementConvertibleSpecInterface(idFactory astmodel.IdentifierFactory) Stage {
```

Comment:
This file is `implement_chained_conversion_interface` but the one with status is `implement_convertible_status_interface` - feels like there should be a pattern that isn't being followed by one of them?